src/vsr/README.md

```diff
@@ -0,0 +1,209 @@
+# Protocols
+
+### Commands
+
+| `vsr.Header.Command` |  Source |       Target | Protocols                                                      |
+| -------------------: | ------: | -----------: | -------------------------------------------------------------- |
+|               `ping` | replica |      replica | [Ping (Replica-Replica)](#protocol-ping-replica-replica)       |
+|               `pong` | replica |      replica | [Ping (Replica-Replica)](#protocol-ping-replica-replica)       |
+|        `ping_client` |  client |      replica | [Ping (Replica-Client)](#protocol-ping-replica-client)         |
+|        `pong_client` | replica |       client | [Ping (Replica-Client)](#protocol-ping-replica-client)         |
+|            `request` |  client |      primary | [Normal](#protocol-normal)                                     |
+|            `prepare` | replica |       backup | [Normal](#protocol-normal), [Repair WAL](#protocol-repair-wal) |
+|         `prepare_ok` | replica |      primary | [Normal](#protocol-normal), [Repair WAL](#protocol-repair-wal) |
```

Comment:
prepare_ok is only for Normal, not Repair WAL, no?

Reply:
The divide between protocols is sort of artificial.. I split them up to make them more understandable. A replica will send a `prepare_ok` after it receives a `prepare` as long as it is in `status=normal`. So it can happen during repair.